### PR TITLE
3DMLoader: Fix typos and reorder imports alphabetically

### PR DESCRIPTION
**Pull request by GitHubDragonFly**
Related issue: None

**Description**

- Fix typos and reorder imports alphabetically

### CODE REVIEW

File: examples/jsm/loaders/3DMLoader.js [hunk 1]
```diff
@@ -226,7 +226,8 @@ class Rhino3dmLoader extends Loader {
 
 			return new MeshStandardMaterial( {
 				color: new Color( 1, 1, 1 ),
-				metalness: 0.8,
+				metalness: 0.1,
+				roughness: 0.6,
```
Comment (Mugen87) [inline examples/jsm/loaders/3DMLoader.js:230]: > Also modify the default MeshStandardMaterial params (this is arbitrary)

Since they are arbitrary, please revert these changes.

File: examples/jsm/loaders/3DMLoader.js [hunk 2]
```diff
@@ -256,11 +257,11 @@ class Rhino3dmLoader extends Loader {
...
 			const pbr = material.pbr;
 
-			mat.anisotropy = pbr.anisotropy;
+			mat.anisotropy = pbr.anisotropic;
 			mat.anisotropyRotation = pbr.anisotropicRotation;
 			mat.color = new Color( pbr.baseColor.r, pbr.baseColor.g, pbr.baseColor.b );
-			mat.clearCoat = pbr.clearCoat;
-			mat.clearCoatRoughness = pbr.clearCoatRoughness;
+			mat.clearcoat = pbr.clearcoat;
+			mat.clearcoatRoughness = pbr.clearcoatRoughness;
```
Comment (Mugen87) [inline examples/jsm/loaders/3DMLoader.js:264]: @fraguada Can you confirm these changes are correct?

File: examples/jsm/loaders/3DMLoader.js (hunk 1 above)
Comment (GitHubDragonFly) [reply examples/jsm/loaders/3DMLoader.js]: It has been reverted.

**Comment by GitHubDragonFly** [comment]
Just to show what this PR is based on, here is the picture of the Rhino3dm Module as logged in the browser console and showing the property names of the PhysicallyBasedMaterial.

Rhino3dm Module (console output)

**Comment by Mugen87** [comment]
Another reference: https://developer.rhino3d.com/api/rhinocommon/rhino.docobjects.physicallybasedmaterial

I guess it's safe to merge then^^.
